packages/docs/docs/guides/architecture.mdx

```diff
@@ -125,27 +161,19 @@ OPENAI_API_BASE='/v1' yarn turbo run dev --scope create-react-app-demo
 See `packages/create-react-app-demo/backend/index.ts` for a sample proxy server.
 
 Alternatively, use [this template repo](https://github.com/fixie-ai/ai-jsx-template-cra-with-vercel-proxy) for an example that uses a [Vercel edge function](https://vercel.com/docs/concepts/functions/edge-functions).
+:::
 
-## UI on the client; AI.JSX on the server
+## Architecture #3: AI.JSX Server-Side + Client UI
 
 :::tip Recommendation
 This is the most stable way to run and deploy an AI.JSX powered app at the moment.
```

Comment:
```suggestion
This is the most stable way to run and deploy an AI.JSX powered app.
```

I probably wrote this originally, but I think docs shouldn't say things like "right now" or "at the moment" – everything implicitly refers to the current moment.

Also, let's add a note to this section about integrating with the Vercel AI SDK. Perhaps it'll reference the larger docs that @petersalas is going to make about that.

Reply:
making the content change you suggest here. however, i am not clear what exactly you want to add about the Vercel AI SDK. is there something more to say now? if so, what? or is this more of a callout to circle back with Peter once his work is done?